string source_robot
swarmus_ros_simulation/Interloc[] otherRobots